iothub/device/src/Transport/Amqp/AmqpUnit.cs

```diff
@@ -869,51 +869,6 @@ await Task
             }
         }
 
-        internal async Task SendTwinMessageAsync(
```

Comment:
Unused method?

Reply:
I think this was a bad merge, actually. I changed it from public to internal and moved it up above the private methods.

Reply:
It's been removed now

Reply:
There's only 1 method by this name now

Reply:
Yeah, I moved it higher (ordering of internal/private) and made a code change.

You'll want to fix the merge please.

Reply:
I'm seeing only 1 method by the name of "SendTwinMessageAsync", so it must have been fixed in one of the subsequent rebases on my branch

Reply:
it is removed here and reappears on 962 on the green side. The contents of the method here are different than down there (slightly) and the order was changed.

I'm just asking that you move the method back up here and maintain the changes.

Reply:
internal above private, got it. done.